w3c.json

```
@@ -0,0 +1,5 @@
+ {
+    "group":      109735
```

I would put the CG id here (87846)
```suggestion
    "group":      87846
```